src/Handler/CurlFactory.php

```diff
@@ -178,6 +178,18 @@ private static function createRejection(EasyHandle $easy, array $ctx): PromiseIn
             \CURLE_GOT_NOTHING          => true,
         ];
 
+        if ($easy->createResponseException) {
+            return \GuzzleHttp\Promise\rejection_for(
```

Comment:
This function is deprecated. You need to use the static class method.

Reply:
@Nyholm It's a shame phpstan is not loaded using composer so that we can enable checking for deprecated calls.

Reply:
Updated - it looks like master had changed since I updated this PR, latest commit should be good.

Reply:
> It's a shame phpstan is not loaded using composer so that we can enable checking for deprecated calls.

Are you sure about that? 

Im not sure that function was deprecated when PHPStan was running on this PR.